charts/prompt-keda/templates/scaled-job.yaml

```diff
@@ -2,6 +2,8 @@ apiVersion: keda.sh/v1alpha1
 kind: ScaledJob
 metadata:
   name: {{ include "prompt-keda.fullname" . }}
+  annotations:
+    argocd.argoproj.io/hook: PostSync
```

Comment:
Overall this makes sense.  Curious did you try to run the init jobs with a `PreSync` hook and this with a `Sync` hook?